requirements_plugins.txt

```diff
@@ -0,0 +1,7 @@
+# Plugins:
+plover-treal==1.0.0
+
+# Dependencies:
+hidapi==0.7.99.post20
```

Comment:
OK, so I added `utils/check_requirements.py`: it's used at the end of the distribution build to check the requirements (and print them, to make it easier to maintain them). This will load the plugins (without suppressing errors), so it can be used to detect some issues earlier (e.g. blacklisting some necessary PyQt5 files).

Note: our AppImage Docker image [needs to be updated](https://github.com/openstenoproject/plover-docker-images/pull/1).